src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -4398,13 +4554,17 @@ public override BoundNode VisitAsOperator(BoundAsOperator node)
                             }
                             break;
 
+                        case ConversionKind.ImplicitNullable:
+                            nullableAnnotation = (_resultType.IsNullableType() && _resultType.NullableAnnotation.IsAnyNullable()) ? NullableAnnotation.Nullable : NullableAnnotation.NotNullable;
+                            break;
+
                         default:
```

Comment:
>default: [](start = 24, length = 8)

Do we need a special handling for conversion to nullable value type from a type parameter constrained to the underlying value type, or to the same nullable value type (can happen with overriding)?

Reply:
Added tests but no code changes.

---
In reply to: [239949105](https://github.com/dotnet/roslyn/pull/31499#discussion_r239949105) [](ancestors = 239949105)